README.md

```diff
@@ -38,10 +38,12 @@ for (let workplace of workplaces) {
 
 * account
   * [`login(email, password)`](docs/account.md#loginemail-password)
+  * [`backofficeLogin()`](docs/account.md##backofficelogin)
```

Comment:
Not the most elegant place for this. Top level resource methods should be extremely simple. There should be only one `login()` method.

Can you refactor the `login` method to accept new arguments?

```js
await deskbookers.account.login({
  email,
  password,
  backoffice: Boolean
})